src/wp-includes/blocks/footnotes.php

```diff
@@ -110,6 +110,19 @@ function wp_add_footnotes_to_revision( $fields ) {
  * @return string The field value.
  */
 function wp_get_footnotes_from_revision( $revision_field, $field, $revision ) {
-	return get_metadata( 'post', $revision->ID, $field, true );
+	$footnotes = json_decode( get_metadata( 'post', $revision->ID, $field, true ) );
+	if ( empty( $footnotes ) ) {
+		return $revision_field;
+	}
+	$footnotes_html = '';
+	$x = 1;
+	foreach( $footnotes as $footnote ) {
```

Comment:
cleaner display extracts the footnote content and adds numbering (needs double checking that order is correct)

Reply:
cc: @ellatrix